release.sh

```diff
@@ -176,6 +188,9 @@ if [[ ! $PUBLISH ]]; then
 
     echo ""
     echo "The automatic parts are done!"
+    echo "Find the url of the deployment for the latest version in"
+    echo "https://app.netlify.com/sites/katex/deploys?filter=master"
+    echo "and add it to 'website/old-versions.json'"
```

Comment:
I cannot find a reliable way to retrieve the url of Netlify deployments for the specific version, so it has to be added manually for now.